### DISCUSSION TITLE
modules missing issue

### DISCUSSION THREAD

**Discussion by ADI711** [pyscript/pyscript#1370] [Q&A] [answered]
I am currently facing the same issue as above and i have tried removing any whitespaces or tabs.
the only difference is i have placed the tag inside the head tag and i still get the same error.
![image](https://user-images.githubusercontent.com/42679100/230745227-f53e9a5f-27a9-4216-ab3a-c021f788c85b.png)
![image](https://user-images.githubusercontent.com/42679100/230745243-58467c79-0908-4302-aba8-4fc4d29a0fc9.png)
I was directed here from https://github.com/pyodide/pyodide
can anyone assist me on this please?

**Comment by JeffersGlass** [comment]
It looks like you're using the \<py-env\> syntax which was present back in early releases of PyScript, but has since between replaced with [\<py-config\>](https://docs.pyscript.net/latest/reference/elements/py-config.html). The corresponding py-config tag would be:

```
<py-config>
  packages = ['pandas', 'sklearn']
  [[fetch]]
  files = ['./movies.csv']
</py-config>
```

**Reply by ADI711** [reply]
thanks @JeffersGlass but after replacing it with <py-config> i am getting a different error
(PY1001): Unable to install package(s) 'pandas,sklearn'. Reason: Can't find a pure Python 3 Wheel for package(s) 'pandas,sklearn'. See: https://pyodide.org/en/stable/usage/faq.html#micropip-can-t-find-a-pure-python-wheel for more information.

**Reply by JeffersGlass** [reply]
`sklearn` is the name you use to import the scikit-learn package in Python, but to install it via pip (or in this case Micropip), the package names is `scikit-learn`

```html
<py-config>
    packages = ['pandas', 'scikit-learn']
</py-config>
<py-script>
    import pandas
    import sklearn
    print("HELLO!")
</py-script>
```

**Reply by ADI711** [reply, marked as answer]
sry for the simple mistake from my side @JeffersGlass thanks
